config/checkstyle-non-main-files-suppressions.xml

```diff
@@ -60,6 +60,8 @@
   <!-- test resources are weird by design, no validations in them -->
   <suppress id="lineLength"
          files="src[\\/]it[\\/]resources[\\/].*[\\/]InputLineLength.*\.java"/>
+  <suppress id="lineLength"
+         files="src[\\/]it[\\/]resources[\\/].*[\\/]InputPackageStatement.*\.java"/>
```

Comment:
`InputPackageStatement.*\.java`
is covering 
`InputPackageStatementLongName.*\.java`
so second is useless.

make it exact `InputPackageStatement\.java`

Reply:
done